app/code/BodenkoVV/AskQuestion/Controller/Adminhtml/Question/MassDelete.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace BodenkoVV\AskQuestion\Controller\Adminhtml\Question;
+
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Backend\App\Action\Context;
+use Magento\Ui\Component\MassAction\Filter;
+use BodenkoVV\AskQuestion\Model\QuestionFactory;
+use BodenkoVV\AskQuestion\Model\ResourceModel\Question\CollectionFactory;
+
+/**
+ * Class MassDelete
+ */
+class MassDelete extends \Magento\Backend\App\Action
+{
+    /**
+     * Authorization level of a basic admin session
+     *
+     * @see _isAllowed()
+     */
+    const ADMIN_RESOURCE = 'BodenkoVV_AskQuestion::delete';
+
+    /**
+     * @var Filter
+     */
+    protected $filter;
+
+    /**
+     * @var CollectionFactory
+     */
+    protected $collectionFactory;
+
+
+    public $_questionFactory;
+
+    /**
+     * @param Context $context
+     * @param Filter $filter
+     * @param CollectionFactory $collectionFactory
+     */
+    public function __construct(
+        Context $context,
+        Filter $filter,
+        CollectionFactory $collectionFactory,
+        QuestionFactory $questionFactory
+    )
+    {
+        $this->filter = $filter;
+        $this->collectionFactory = $collectionFactory;
+        $this->_questionFactory = $questionFactory;
+        parent::__construct($context);
+    }
+
+    /**
+     * Execute action
+     *
+     * @return \Magento\Backend\Model\View\Result\Redirect
+     * @throws \Magento\Framework\Exception\LocalizedException|\Exception
+     */
+    public function execute()
+    {
+        $collection = $this->filter->getCollection($this->collectionFactory->create());
+        $collectionSize = $collection->getSize();
+
+        foreach ($collection as $page) {
+
+            $collectionFactory = $this->_questionFactory->create();
```

Comment:
$this->_questionFactory->create(); will return Question, but not collection of questions. Please check and rename the variable accordingly

Reply:
+